File: src/prime.py
#! /usr/bin/env python2
import sys

if len(sys.argv)<2:
    print "Requires one argument, remember target"
    sys.exit()

import math
def isPrime(n):
    if n%2==0:
        return False
    else:
        for e in range(3,int(math.sqrt(n)+1),2):
            if n%e==0:
                return False
        return True
s=0
target=int(sys.argv[1])

for i in range(2,target):
    if isPrime(i):
        s+=1
        #print (i)
print s

File: src/prime.js
#! /usr/bin/env node

if(process.argv.length<3){
	console.log("Requires one argument, remember target");
	process.exit(0);
}

function isPrime(n){
	if(i<2) return false;
	if(i==2) return true;
	if(n%2==0) return false;
	for(var i=3;i<Math.sqrt(n)+1;i+=2){
		if(n%i==0) return false;
	}
	return true;
}

var sum=0, target=parseInt(process.argv[2]);;

for(var i=2;i<target;i++){
	if(isPrime(i)){
		sum++;
		//console.log(i);
	}
}

console.log(sum);
